src/services/cv_service.py

```diff
@@ -0,0 +1,153 @@
+import PyPDF2
+from datetime import date
+
+
+# Returns the first name from inside the cv
+
+
+def get_first_name_from_cv(cv_content: str) -> str:
+    all_lines = cv_content.split('\n')
+    specific_line: str
+
+    for line in all_lines:
+        if 'Name:' in line:
+            specific_line = line
+
+    first_name = specific_line.split(' ')[1]
+
+    return first_name
+
+
+# Returns the last name from inside the cv
+
+
+def get_last_name_from_cv(cv_content: str) -> str:
+    all_lines = cv_content.split('\n')
+    specific_line: str
+
+    for line in all_lines:
+        if 'Name:' in line:
+            specific_line = line
+
+    last_name = specific_line.split(' ')[2]
+
+    return last_name
+
+
+# Returns the date of birth from inside the cv
+
+
+def get_date_of_birth_from_cv(cv_content: str) -> date:
+    all_lines = cv_content.split('\n')
+    specific_line: str
+
+    for line in all_lines:
+        if 'Geburtsdatum / -ort:' in line:
```

Comment:
Switch this to english please:
```suggestion
        if 'Date / Place of Birth:' in line:
```
That is how it is called in one of our generated cvs.

Reply:
Please also switch 'Anschrift', 'E-Mail Adresse' and 'Telefonnummer'

Reply:
Updated all methods